packages/components/src/RecurringSelect/components/MonthlyDayOfWeekSelect/MonthlyDayOfWeekSelect.css

```
@@ -0,0 +1,32 @@
+.container {
+  display: flex;
+  margin-bottom: calc(-1 * var(--border-base));
+}
```

can do

```suggestion
.container {
  display: grid;
  grid-template-columns: var(--space-largest) 1fr;
  align-items: center;
  gap: var(--space-small);
  margin-bottom: calc(-1 * var(--border-base));
}
```

and you won't have to add the last 2 css class and probably cut down the amount of nth-child you have there